### ISSUE TITLE
[NEWTHEME] Name of your theme

### ISSUE THREAD

**Issue by BlueFalconHD** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#151]
```
{
		"title": "SimpleFox Feather Edition",
		"link": "https://github.com/BlueFalconHD/SimpleFox-Feather/",
		"description": "A fork of SimpleFox that uses feather icons!",
		"image": "https://github.com/BlueFalconHD/SimpleFox-Feather/raw/master/assets/head.png"
}
```

**Comment by BlueFalconHD** [comment]
![](https://github.com/BlueFalconHD/SimpleFox-Feather/raw/master/assets/head.png)

**Comment by andreasgrafen** [comment]
There currently are no changes to the CSS in any way. So as of right now this will be exactly the same as the original SimpleFox which is already in our database.
Maybe you forgot to commit the changes to the actual userChrome file? o:

**Comment by BlueFalconHD** [comment]
> There currently are no changes to the CSS in any way. So as of right now this will be exactly the same as the original SimpleFox which is already in our database.
> Maybe you forgot to commit the changes to the actual userChrome file? o:

Yep. I did. I also did change SimpleFox a lot. I changed icons in the icons section at the bottom.

**Comment by BlueFalconHD** [comment]
I forgot to commit 🤦

**Comment by BlueFalconHD** [comment]
I will commit later today @andreasgrafen

**Comment by andreasgrafen** [comment]
No worries! Happends to the best of us. :D

**Comment by BlueFalconHD** [comment]
Done! @andreasgrafen

**Comment by andreasgrafen** [comment]
You don't have issues enabled for the repo, so I'll leave this here for now:
Hotlinking the icons from GitHub seems like a liability as this could cause issues with both scenarios in which the user has a slow connections or is completely offline. I would suggest to bundle the SVGs with your userChrome to avoid this. o:

**Comment by BlueFalconHD** [comment]
working on it. I had a science project for school, so I didn't have much time.

**Comment by andreasgrafen** [comment]
That's absolutely fine! There is no need to rush anything. School is more important; so take your time. ^—^

**Comment by BlueFalconHD** [comment]
@andreasgrafen done!

**Comment by andreasgrafen** [comment]
Awesome. Not it only needs some tags, and everything should be good to go. :3
I would suggest something along the lines of `['Simplefox', 'Fork', 'Icons', 'BlueFalconHD']`, would that be okay?

**Comment by BlueFalconHD** [comment]
yes!

**Comment by andreasgrafen** [comment]
Done and dusted. ^—^
